src/C4Update.cpp

```diff
@@ -786,8 +811,9 @@ bool C4UpdatePackage::MkUp(C4Group *pGrp1, C4Group *pGrp2, C4GroupEx *pUpGrp, bo
 			extern const char **C4Group_SortList;
 			UpdGroup.SortByList(C4Group_SortList, ChildGrp2.GetName());
 			UpdGroup.Close(false);
-			// check entry times
-			if (!pGrp1 || (pGrp1->EntryTime(strItemName) != pGrp2->EntryTime(strItemName)))
+			// always add the entire group if mising targets are allowed
+			// otherwise check entry times
+			if (AllowMissingTarget || !pGrp1 || (pGrp1->EntryTime(strItemName) != pGrp2->EntryTime(strItemName)))
```

Comment:
If possible, it would be more logical (for the meaning of `Modified`) to move the `AllowMissingTarget` check to the `if` below.
Alternatively, rename `Modified` to `IncludeInUpdate` or similar.

Reply:
> If possible, it would be more logical (for the meaning of Modified) to move the AllowMissingTarget check to the if below.

That would unnecessarily call `EntryTime`.